frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -90,64 +77,39 @@ export function FeatureFlag(): JSX.Element {
-                    layout="vertical"
-                    form={form}
-                    initialValues={{ name: featureFlag.name, key: featureFlag.key, active: featureFlag.active }}
-                    onValuesChange={(newValues) => {
-                        if (featureFlagId !== 'new' && newValues.key) {
-                            setHasKeyChanged(newValues.key !== featureFlag.key)
-                        }
-                        setFeatureFlag({ ...featureFlag, ...newValues })
-                    }}
-                    onFinish={(values) =>
-                        saveFeatureFlag({
-                            ...featureFlag,
-                            ...values,
-                            filters: featureFlag.filters,
-                        })
-                    }
-                    requiredMark={false}
-                    scrollToFirstError
+                    logic={featureFlagLogic}
+                    props={logicProps}
+                    formKey="featureFlag"
+                    className="ant-form-vertical ant-form-hide-required-mark"
+                    // scrollToFirstError
                 >
                     <PageHeader
                         title="Feature Flag"
                         buttons={
                             <div style={{ display: 'flex' }}>
-                                <Form.Item className="enabled-switch">
-                                    <Form.Item
-                                        shouldUpdate={(prevValues, currentValues) =>
-                                            prevValues.active !== currentValues.active
-                                        }
-                                        style={{ marginBottom: 0, marginRight: 6 }}
-                                    >
-                                        {({ getFieldValue }) => {
-                                            return (
-                                                <span className="ant-form-item-label" style={{ lineHeight: '1.5rem' }}>
-                                                    {getFieldValue('active') ? (
-                                                        <span className="text-success">Enabled</span>
-                                                    ) : (
-                                                        <span className="text-danger">Disabled</span>
-                                                    )}
-                                                </span>
-                                            )
-                                        }}
-                                    </Form.Item>
-                                    <Form.Item name="active" noStyle valuePropName="checked">
-                                        <Switch />
-                                    </Form.Item>
-                                </Form.Item>
-                                {featureFlagId !== 'new' && (
+                                <Field name="active">
```

Comment:
I think I'd approve just for the change of language from `Form.Item` to `Field` 🥇